rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -168,6 +172,7 @@ public void startNode(String[] args) throws Exception {
         }
 
         messageHandler.start();
+        txHandler.start();
```

Comment:
Same as before: `getNodeMessageHandler` depended on `TxHandler`, so `txHandler.start()` should go before `messageHandler.start()`